source/broadband/webpa_internal.c

```diff
@@ -73,7 +73,6 @@ char *objectList[] ={
 "Device.LogBackup.",
 "Device.IoT.",
 "Device.NotifyComponent.",
-"Device.LogAgent.",
 "Device.X_RDKCENTRAL-COM_Webpa.",
```

Comment:
If we are removing an entry from objectList , then object level1 count also need to modified accordingly here line 37 and line 40,
https://github.com/xmidt-org/parodus2ccsp/blob/4a73f734e0c3b609f2567817a5824b0a13ec7083/source/broadband/include/webpa_internal.h#L37